src/testutils.jl

```diff
@@ -293,7 +293,7 @@ function test_range_evaluation(d::DiscreteUnivariateDistribution)
     rmin = int(islowerbounded(d) ? vmin : quantile(d, 0.001))::Int
     rmax = int(isupperbounded(d) ? vmax : quantile(d, 0.999))::Int
 
-    p0 = pdf(d, [rmin:rmax])
+    p0 = pdf(d, [rmin:rmax;])
```

Comment:
`[itr]` no longer collects on 0.4. The idiomatic way is now `collect(itr)`, but `[itr;]` (which I believe is meant for cases like `[a; b; itr; c;]`) works, too.